backbone/src/main/java/org/researchstack/backbone/ui/step/layout/PasscodeCreationStepLayout.java

```diff
@@ -0,0 +1,208 @@
+package org.researchstack.backbone.ui.step.layout;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.os.Handler;
+import android.support.v4.content.ContextCompat;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+
+import com.jakewharton.rxbinding.widget.RxTextView;
+
+import org.researchstack.backbone.R;
+import org.researchstack.backbone.StorageAccess;
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.PasscodeStep;
+import org.researchstack.backbone.step.Step;
+import org.researchstack.backbone.ui.callbacks.StepCallbacks;
+import org.researchstack.backbone.ui.views.PinCodeLayout;
+import org.researchstack.backbone.utils.ThemeUtils;
+
+public class PasscodeCreationStepLayout extends PinCodeLayout implements StepLayout
```

Comment:
This is essentially the same class of SignUpPincodeCreationStepLayout; however, I want to deprecate that moving forward, as this will become the passcode creation layout.